README.md

```diff
@@ -151,6 +151,34 @@ the `name` will be used. If none of them are defined, it's an error.
 
 `kind` must be one of the kinds defined in `kubectl get`.
 
+## Configuration defaults
+
+CLM will look for a `config-defaults.yaml` file in the cluster configuration
+directory. If the file exists, it will be evaluated as a Go template with all
+the usual CLM variables and functions available, and the resulting output will
+be parsed as a simple key-value map. CLM will use the contents of the file to
+populate the cluster's configuration items, taking care not to overwrite the
+existing ones.
+
+For example, you can use the defaults file to have different settings for
+production and test clusters, while keeping the manifests readable:
+
+* **config-defaults.yaml**:
+    ```yaml
+    {{ if eq .Environment "production"}}
+    autoscaling_buffer_pods: "3"
+    {{else}}
+    autoscaling_buffer_pods: "0"
+    {{end}}
```

Comment:
What do you think about dedicated config files for each environment, like other projects do with `development.yaml`, `production.yaml` etc. In our case:`${cluster.Environment}.yaml`. That would lead to some duplication between the files but would get rid of the `if/else` logic in the config file.

Reply:
In my opinion a simple `if/else` is better than repeating the same default in multiple config files, but we could also have something like `config-defaults/global.yaml`. I would wait and see if we start abusing the templating, and then implement some better solutions. WDYT?

Reply:
Let's wait then. A `global.yaml` that the individual envs would "inherit" from would be overkill for now.